package.json

```diff
@@ -56,11 +56,8 @@
     "test:ci:vitest:unit": "vitest run --coverage tests/unit/",
     "test:ci:vitest:integration": "vitest run --coverage tests/integration/",
     "e2e": "node ./tools/e2e/run.js",
-    "docs": "npm run --prefix=site build",
+    "site:build": "npm run build && npm install --prefix=site --no-audit && npm run --prefix=site build",
```

Comment:
I don't love the `npm install` inside a script but that's a whole can of worms that didn't seem worth pursuing